buildpack.toml

```
@@ -46,6 +46,11 @@ name        = "BP_NATIVE_IMAGE_BUILD_ARGUMENTS"
 description = "arguments to pass to the native-image command"
 build       = true
 
+[[metadata.configurations]]
+name        = "BP_NATIVE_IMAGE_BUILD_FROM_JAR"
```

```suggestion
name        = "BP_NATIVE_IMAGE_BUILD_JAR"
```

How about this? Shortens it a bit and I think it still matches the description.

---

Done